src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -215,12 +215,14 @@ namespace DataHeatBalance {
     extern int const MixingSourceZonesOnly;
     extern int const AllZones;
 
-    // Parameter for zone air flow mass balancing method
-    extern int const AdjustMixingOnly;
-    extern int const AdjustReturnOnly;
-    extern int const AdjustMixingThenReturn;
-    extern int const AdjustReturnThenMixing;
-    extern int const NoAdjustReturnAndMixing;
```

Comment:
Deleted and substituted with Enum class.

Reply:
:+1: